### ISSUE TITLE
Un expected results

### ISSUE THREAD

**Issue by aimanmutasem** [bentrevett/pytorch-seq2seq#95]
Hi, ben :)
When I execute notebook 6 (Attention is All You Need) I have got lower results compare to yours, as follow : 

| Test Loss: 1.674 | Test PPL:   5.332 |
BLEU score = 35.35 

The original results are :
| Test Loss: 1.662 | Test PPL:   5.268 |
BLEU score = 36.09

I have changed one single line [device = torch.device('cuda' if torch.cuda.is_available() else 'cpu')] to [device = torch.device('cuda:0')]. 

What is your opinion, and where I missed?

**Comment by bentrevett** [comment]
Due to random variations in the training, you will most probably not get the exact same results as those in the tutorials. Even if you use the same seed as the tutorials if you are using a different version of PyTorch, or even the same version of PyTorch installed on a different platform, you will not get the exact same results.

The results in this tutorial are only taken over a single seed, and sometimes I may have gotten lucky and found a decent seed. In reality, we should always run each experiment with multiple seeds, say 3-5, and then take an average of these.

**Comment by aimanmutasem** [comment]
Thank you Bin, :)

For notebook 5, I have got 33.98 BLEU score with (+0.69‬) improve :)
